### ISSUE TITLE
mako.co.il: breakage

### ISSUE THREAD

**Issue by Hebrew-uBO-User** [uBlockOrigin/uAssets#24915]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://mobile.mako.co.il/
https://www.mako.co.il/
```

### Description

**Crash** on Firefox for **Android** (default settings + EasyList **Hebrew**).
Issue does not occur if uBO is disabled.
What do you suggest?

There is another issue with this site which isn't caused by uBO.
On desktop devices, when you click on a link with the middle mouse button, the link is opened both in a new tab and in the same tab.
Can it be fixed with uBO?

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml


```
</details>

**Comment by stephenhawk8054** [comment]
Don't remove the troubleshooting information.

**Comment by Hebrew-uBO-User** [comment]
Please re-open this issue.

@stephenhawk8054 
First issue (does not occur if uBO is disabled):
```
  Firefox for Android 129.0 (latest stable release) with default settings
+ uBlock Origin 1.59.0 (latest stable release) with default settings 
+ EasyList Hebrew.
No other extensions are used.
```

Second issue (not caused by uBO):
```
  Firefox 129.0.1 (latest stable release) with default settings
+ uBlock Origin 1.59.0 (latest stable release) with default settings 
+ EasyList Hebrew.
No other extensions are used.
```

@ghajini 
@MasterKia 
Use Firefox for Android and go to `https://mobile.mako.co.il/` or `https://www.mako.co.il/`
Wait 2 minutes, then scroll up & down the page, Firefox will become very slow and eventually crash.

What about the second issue?
Go to `https://www.mako.co.il/` and click on any link with the mouse middle button.
Can you reproduce?

Thank you

**Comment by stephenhawk8054** [comment]
The troubleshooting information is not just for knowing which lists you are using, but also for checking which connections are being blocked, are there any excessive blocking connections occur or not...? Sometimes these happen depending on the region.

**Comment by Hebrew-uBO-User** [comment]
Very weird...
I can reproduce the first issue on Opera for Android with built-in adblocker enabled. Issue does not occur if the built-in adblocker is disabled.
I can reproduce the second issue on different browsers (Firefox, Chromium) on different machines (with or without uBO enabled, with or without a VPN).

@sha-265 
Can you reproduce?

@ghajini 
@stephenhawk8054 

```
uBlock Origin: 1.59.0
Firefox: 129
filterset (summary):
 network: 154231
 cosmetic: 49010
 scriptlet: 20723
 html: 1928
listset (total-discarded, last-updated):
 added:
  ISR-0: 1910-3, 2m
 default:
  user-filters: 0-0, never
  ublock-filters: 39853-125, 2m
  ublock-badware: 10442-6, 2m
  ublock-privacy: 1059-4, 2m
  ublock-unbreak: 2556-1, 1m Δ
  ublock-quick-fixes: 159-4, 2m
  easylist: 85251-180, 2m
  easyprivacy: 53030-65, 2m
  urlhaus-1: 29620-0, 2m
  plowe-0: 3547-993, 2m
filterset (user): [array of 11 redacted]
trustedset:
 added: [array of 6 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 372 ms
 maxAssetCacheWait: 201 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 4
 network:
  doubleclick.net: 1
  googletagmanager.com: 2
  outbrain.com: 1
 extended:
  ##.OUTBRAIN
  ##.adunitContainer
  ##.adBox
  ##.h033Desktop_root__UovNL
  ##.v025Desktop_root__bxIn4
  ##[href^="https://www.holidayfinder.co.il/"][href*="?utm_source=…
  ##[class^="Ad_root_"]
  ##[class^="Layout_premiumWrapperElement_"]
  ##+js(abort-on-property-write, upManager)
```

**Comment by stephenhawk8054** [comment]
@Hebrew-uBO-User So, is your first issue on mobile or desktop? Your configuration shows the desktop version.

**Comment by Hebrew-uBO-User** [comment]
@stephenhawk8054 
The first issue is on mobile (crash after a few minutes - caused by adblocking).
The second issue is on desktop (links are being opened in the same tab when clicking on middle mouse button, this is **not** caused by adblocking).

I added troubleshooting information for the second issue.

**Comment by stephenhawk8054** [comment]
Yeah, I mean get the troubleshooting information for the first issue.

**Comment by rerere123123** [comment]
This issue is back again, on Windows Firefox and iOS Safari.
For example: https://www.mako.co.il/mako-vod-live-tv/VOD-6540b8dcb64fd31006.htm

**Comment by sha-265** [comment]
> This issue is back again, on Windows Firefox and iOS Safari.
> For example: https://www.mako.co.il/mako-vod-live-tv/VOD-6540b8dcb64fd31006.htm

No problem in FF on Windows.

**Comment by rerere123123** [comment]
Yeah, there is no issue on Firefox. My bad, I was referring to another website.
However, it still detects uBOL on iOS, and gives instructions on how to disable it.
For example: https://www.mako.co.il/mako-vod-live-tv/VOD-6540b8dcb64fd31006.htm

*Other mako.co.il pages such as https://www.mako.co.il/ don't work properly and render a completely blank page, but don’t show a message about it detecting uBOL.

<details>

<img width="1206" height="2456" alt="Image" src="https://github.com/user-attachments/assets/698e57c7-1ab9-4a53-98a1-ddd1e9c77991" />

</details>
